journal/202307.md

```diff
@@ -0,0 +1,596 @@
+- _Hint: Read the Guidelines on how to make DJ great: https://github.com/xaur/decred-news/blob/docs/guidelines.md_
+
+}
+
+{ _upload title image xxxxxx.1_ }
+
+_Image: { "Title by @author. Description..." OR just "Description..." }_
+
+{ _Month's highlights, no links here, 5-7 bullets max._
+
+- 
+
+- 
+
+- 
+
+- 
+
+- 
+
+}
+
+Contents: { _Sync ToC at the very end, when all text is finished_ }
+
+- [Development](#development)
+- [People](#people)
+- [Governance](#governance)
+- [Network](#network)
+- [Ecosystem](#ecosystem)
+- [Outreach](#outreach)
+- [Events](#events)
+- [Media](#media)
+- [Markets](#markets)
+- [Relevant External](#relevant-external)
+
+
+## { Some Announcement }
+
+{ _Add individual sections for high-impact announcements, such as consensus votes, software releases, milestones, major bugs, etc._ }
+
+<a id="development" />
+
+## Development
+
+The work reported below has the "merged to master" status unless noted otherwise. It means that the work is completed, reviewed, and integrated into the source code that advanced users can [build and run](https://medium.com/@artikozel/the-decred-node-back-to-the-source-part-one-27d4576e7e1c), but is not yet available in release binaries for regular users.
+
+
+### dcrd
+
+_[dcrd](https://github.com/decred/dcrd) is a full node implementation that powers Decred's peer-to-peer network around the world._
+
+The following work has been merged in `master` towards future releases:
+
+- Updated `GetBlock` and `GetBlockHeader` commands to show [the new proof of work hash](https://github.com/decred/dcrd/pull/3154). For blocks after [DCP-11: Change PoW to BLAKE3 and ASERT](https://github.com/decred/dcps/blob/master/dcp-0011/dcp-0011.mediawiki) activates, the *proof of work hash* will be the new BLAKE3 hash. For blocks prior to DCP-11, the *block hash* and the *proof of work hash* will be identical. This update enables consumers like [dcrdata](#dcrdata) to easily access the new hashes.
+- Updated the [Docker image](https://github.com/decred/dcrd/pull/3158) to build the dcrd binary with newer Go and Alpine Linux. Node admins can consider Docker a good security option as opposed to using a virtual machine which has a much bigger attack surface.
+- Updated the RPC server to [dynamically reload new RPC certificates](https://github.com/decred/dcrd/pull/3153) without shutting down and restarting. This is a quality-of-life improvement for node admins, since it enables them to hot-swap the server certificate/key pair, as well as any potential client certificates. For an example, certificate update is required when the RPC server is exposed publicly and its IP address or domain name changes. Another example is when client certificates need to be modified to revoke clients, add new ones, or allow existing clients to change their private keys. This update is robust, minimizes disk access, does not require platform-specific dependencies, and is resilient against user-errors to avoid breaking a working config.
```

Comment:
```suggestion
- Updated the RPC server to [dynamically reload new RPC certificates](https://github.com/decred/dcrd/pull/3153) without shutting down and restarting. This is a quality-of-life improvement for node admins, since it enables them to hot-swap the server certificate/key pair, as well as any potential client certificates. For example, a certificate update is required when the RPC server is exposed publicly and its IP address or domain name changes. Another example is when client certificates need to be modified to revoke clients, add new ones, or allow existing clients to change their private keys. This update is robust, minimizes disk access, does not require platform-specific dependencies, and is resilient against user-errors to avoid breaking a working config.
```

Reply:
Added manually, thanks.